courses/serializers.py

```diff
@@ -0,0 +1,42 @@
+import json
+
+from rest_framework import serializers
+
+from .models import Course, Module
+
+
+class JsonListField(serializers.Field):
+    def to_representation(self, value):
+        if isinstance(value, unicode) or isinstance(value, str):
+            try:
+                value = json.loads(value)
+            except ValueError:
+                raise serializers.ValidationError("Must provide valid JSON.")
+
+        if value is not None and not isinstance(value, list):
```

Comment:
It looks like JSONField disallows null by default, so we may want to remove the first part of the clause